src/App.js

```diff
@@ -2,19 +2,27 @@ import React from "react";
 import { Route, Switch } from "react-router-dom";
 import { Home } from "./Home/Home";
 import { Header } from "./Header/Header";
+import { Footer } from "./zFooter/Footer";
 
 function App() {
   return (
     <div className="App">
-      <Header />
-      <Switch>
-        <Route exact path="/">
-          <Home />
-        </Route>
-        <Route path="*">
-          <div>Whatever, dude</div>
-        </Route>
-      </Switch>
+      <div className="content-wrap">
```

Comment:
Don't call this class `content-wrap`. Come up with something more descriptive of what this element is doing. Also, why is this class a magic string, instead of a SCSS Module?

Reply:
It didn't make sense to me to create a module for only one class. The idea is to have a wrapper holding the entire layout of the page.

Reply:
It's still best for you to use the SCSS modules. You keep things consistent that way. Otherwise, you're not taking the benefits of CSS Modules, and putting yourself at risk of having colliding CSS class names. Mixing normal class names with CSS Modules can increase the confusion of the code base, because of the mixture, and the fact that you're dropping the CSS Module protections you get in the places where you don't use it.

Reply:
🆗